### PR TITLE
capstone project module 1

### PR DESCRIPTION
**Pull request by BouaikIlham**
- [ ] [no linter errors](https://github.com/microverseinc/linters-config).
- [ ] used correct [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md).
- [ ]  I documented my work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
- [ ] In this capstone project I was building a caritative website based on an online e learning web Site build with Html Css JavaScript follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015),including colors. font family font size ...
- [ ] Here's the link to a little video I just made describing some little features about the project: [link](https://www.loom.com/share/0940b9aa934248ba9ab8b1b688487107)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+            The best way to create an amazing learning experience <br> is to define the scope of the training and clearly communicate this to the learner.
+        </h1>
+        <div class="ideal">
+          <span>
+            Learning is the process of acquiring knowledge <strong>skills</strong>  and <strong>values</strong>. Learning needs to be planned and evaluated. It can be measured, and its effectiveness can be assessed. <br>
+            A quality system needs to be in place that monitors and evaluates the learning process. The same goes for eLearning.
+          </span>
+          <h3 class="work-hour">8am - 5pm(MON) ~ (FRI) 8pm</h3>
+          “E-Learning doesn't just "happen"! ...
+        </div>
```
Comment (gemmen29) [inline index.html]: - [x] Kindly try to match the Behance design in the intro section on the home page mobile version by increasing the `font-size` of the heading and work hours

| Your design | Behance design |
| --- | --- | 
| image | image |

- [x] and also increase `font-size` in desktop version

| Your design | Behance design |
| --- | --- | 
| image | image |

File: index.html [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+            <div class="right">
+              Get oppotunities to network with GC affiliates around the globe.
+            </div>
+          </div>
+        </div>
+        <div class="join-btn">
+          <a href="#">SEE THE WHOLE PROGRAM</a>
+        </div>
+      </section>
+      <section id="sponsor" class="sponsor"></section>
```
Comment (gemmen29) [inline index.html]: - [x] Kindly try to match the Behance design in the `featured speakers` section on the home page mobile version by increasing the `image size` of the speaker and making the text start from the beginning.

| Your design | Behance design |
| --- | --- | 
| image | image |

- [x] and also the same in desktop version

| Your design | Behance design |
| --- | --- | 
| image | image |

File: index.html [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+            <img src="./images/google.png" alt="img2" />
+          </figure>
+          <figure>
+            <img src="./images/naver.png" alt="img3" />
+          </figure>
+          <figure>
+            <img src="./images/airbnb.jpg" alt="img4" />
+          </figure>
+        </div>
+      </section>
```
Comment (gemmen29) [inline index.html]: - [x] Kindly try to match the Behance design in the `partners` section on the home page mobile version by increasing the `image size` of each brand, increasing top spacing, and decreasing button spacing.

| Your design | Behance design |
| --- | --- | 
| image | image |

- [x] and also the same in the desktop version

| Your design | Behance design |
| --- | --- | 
| image | image |

File: index.html [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+      <div class="last">
+        <img src="./images/creative-commons-license-logo.jpeg" alt="logo" />
+        <div>
+          <p>&copy; 2022 E-learning. Some Rights Reserved.</p>
+          <p>Created by Ilham Bouaik</p>
+        </div>
+      </div>
```
Comment (gemmen29) [inline index.html:167]: - [x] Kindly try to match the Behance design in the `footer` section on the home page mobile version by increasing the top and bottom spacing and increasing the gap between the logo and text.

| Your design | Behance design |
| --- | --- | 
| image | image |

- [x] and in the desktop version decrease the logo size a little bit.
- [x] Kindly change the `div` tag to be `footer` tag for semantic purposes.

File: about.html [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+    </section>
+    <div class="about-last">
+      <div class="about-last-container">
+        <img src="./images/creative-commons-license-logo.jpeg" alt="logo" />
+        <div>
+          <p>&copy; 2022 creative-commons-license-logo. Some Rights Reserved.</p>
+          <p>Created by Ilham Bouaik</p>
+        </div>
+      </div>
+    </div>
```
Comment (gemmen29) [inline about.html]: - [x] Kindly try to match the Behance design in the `footer` section on the about page (same footer as on the home page) 
- [x] Please make the footer light in the mobile version and dark in the desktop version as in the Behance design.
- mobile version:
image
- desktop version:
image

- [x] Kindly change the `div` tag to be `footer` tag for semantic purposes.

File: about.html [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+        So basically, without learning objectives you won’t know why you’re creating the
+        course, what content to include, what activities to choose, or how to gauge the success of your course.
+        Seems pretty important when you put it that way, doesn’t it?
+      </div>
+      <div class="contact">
+        <p>
+          Please contact us per Email for any further questions about E-learning .
+        </p>
+        <p>
+          <strong><a href="ibouaik@gmail.com">ibouaik@gmail.com</a></strong>
```
Comment (gemmen29) [inline about.html:93]: - [x] Kindly try to match the Behance design in the intro section on the home page mobile version by increasing the `font-size` of the heading and making the text in the center

| Your design | Behance design |
| --- | --- | 
| image | image |

- [x] and also the same in the desktop version

File: README.md [hunk 7]
```diff
@@ -1 +1,50 @@
-# First-Capstone-Project
\ No newline at end of file
+# First-Capstone-Project
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# First-Capstone-Project
+
```
Comment (gemmen29) [inline README.md:13]: - [OPTIONAL] Please add a screenshot of your project in the README file to make your README more professional.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @BouaikIlham, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 8]
```diff
@@ -0,0 +1,176 @@
...
+            <img src="./images/google.png" alt="img2" />
+          </figure>
+          <figure>
+            <img src="./images/naver.png" alt="img3" />
+          </figure>
+          <figure>
+            <img src="./images/airbnb.jpg" alt="img4" />
+          </figure>
+        </div>
+      </section>
```
Comment (VICTOR-CHUKWU) [inline index.html]: - [x] Your partner section on mobile is overflowing horizontally, I think this is as a result of the CSS style you gave your class `partner-container figure` a width of 200% and also `.partner img` width of 150px, kindly use media query to fix this
image

File: about.html [hunk 9]
```diff
@@ -0,0 +1,146 @@
...
+    </section>
+    <footer class="about-last">
+      <div class="about-last-container">
+        <img src="./images/creative-commons-license-logo.jpeg" alt="logo" />
+        <div>
+          <p>&copy; 2022 creative-commons-license-logo. Some Rights Reserved.</p>
+          <p>Created by Ilham Bouaik</p>
+        </div>
+      </div>
+    </footer>
```
Comment (VICTOR-CHUKWU) [inline about.html:141]: - [x] Just as the previous reviewer mentioned, kindly fix the background colour of your footer, it should be dark on desktop and light on mobile, currently, it is light on all devices, use media query to achieve this
your current design 👇 
   
image

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @BouaikIlham , 
## STATUS: REQUIRED CHANGES
Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @BouaikIlham ,
# STATUS: APPROVED! 🥇 💯 
done

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**